src/constants/shaders/colorizeUI.frag

```diff
@@ -45,6 +47,15 @@ uvec4 getUintFromTex(usampler2D tex, int index) {
   ivec2 featurePos = ivec2(index % width, index / width);
   return texelFetch(tex, featurePos, 0);
 }
+
+uint getId(ivec2 uv) {
+  uint rawId = texelFetch(srcTexture, uv, 0).r;
+  if (rawId == 0u) {
+    return 0u;
+  }
+  return rawId + idOffset;
+}
+
```

Comment:
nit: pr would be cleaner if this function were still up above where it used to be